### PR TITLE
thumbor 6.2.0 (new formula)

### PR DESCRIPTION
**Pull request by kyrios**

### CODE REVIEW

File: Formula/thumbor.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  desc "Smart imaging service with on-demand crop, resizing and flipping"
+  homepage "http://thumbor.readthedocs.io"
+  url "https://files.pythonhosted.org/packages/44/1f/5b0c3bcc760b01653e442ce3f1706652ecdf7e644f7979779fd33d450184/thumbor-6.2.0.tar.gz"
+  sha256 "3c569dccd663f6adb6eaee01f8bc8158dd8e64714c40a64679a9ec2700aab923"
+
+  depends_on :python
+  depends_on "homebrew/dupes/zlib"
+  depends_on "homebrew/science/opencv"
+  depends_on "homebrew/python/pillow"
```
Comment (MikeMcQuaid) [inline Formula/thumbor.rb]: Can't do cross-tap dependencies in Homebrew/core I'm afraid.

File: Formula/thumbor.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
+class Thumbor < Formula
+  include Language::Python::Virtualenv
+
+  desc "Smart imaging service with on-demand crop, resizing and flipping"
+  homepage "http://thumbor.readthedocs.io"
+  url "https://files.pythonhosted.org/packages/44/1f/5b0c3bcc760b01653e442ce3f1706652ecdf7e644f7979779fd33d450184/thumbor-6.2.0.tar.gz"
+  sha256 "3c569dccd663f6adb6eaee01f8bc8158dd8e64714c40a64679a9ec2700aab923"
+
+  depends_on :python
+  depends_on "homebrew/dupes/zlib"
```
Comment (MikeMcQuaid) [inline Formula/thumbor.rb]: This ships with macOS.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by woodruffw** [comment]
Any update here @kyrios?

**Comment by kyrios** [comment]
Sorry. I’m a little confused by the answers and didn’t have time yet to google what they mean :)

**Comment by MikeMcQuaid** [comment]
Passing on this but homebrew/science may accept it.

**Pull request closed without merging**